File: Submissions/Codechef.py
# https://www.codechef.com/problems/TALLER
tcs = int(input())
for tc in range(tcs):
    x, y = map(int, input().split())
    print("A" if x > y else "B")

# https://www.codechef.com/problems/PRESENTS
tcs = int(input())
for tc in range(tcs):
    n = int(input())
    print((n // 5) * 4 + (n % 5))

File: Submissions/Codechef.js
// https://www.codechef.com/problems/TALLER
let fs = require("fs");
let data = fs.readFileSync(0, "utf-8");
let idx = 0;
data = data.split("\n");
function input() {
  return data[idx++];
}

let tcs = parseInt(input());
for (let tc = 0; tc < tcs; tc++) {
    let xy = input().split(" ");
    console.log((xy[0] > xy[1] ? "A" : "B"));
}
